land_claims.civmap.json

```diff
@@ -98,27 +98,27 @@
+{"name":"Daedalus\n(Amyr)","shortname":"(Amyr)","color":"#c02237","declutter":3,"id":"f4db782d-7853-4f2b-a629-fcc8f294dd4b","polygon":[[[5860,6411],[6233,6424],[6441,6310],[6499,6231],[6544,6009],[6723,5892],[6840,5956],[6889,5947],[6937,5893],[6929,5864],[6939,5822],[6971,5763],[7001,5730],[7001,5684],[6978,5670],[6954,5661],[6939,5629],[6966,5581],[6967,5542],[6953,5510],[6955,5493],[6938,5471],[6967,5449],[6963,5435],[6944,5423],[6923,5395],[6875,5379],[6875,5358],[6807,5324],[6789,5304],[6789,5210],[6738,5153],[6629,5138],[6583,5178],[6524,5180],[6456,5115],[6438,5051],[6417,5060],[6384,5054],[6366,5043],[6329,5054],[6298,5176],[6218,5218],[6142,5352],[5910,5352],[5856,5442],[5808,5592],[5808,5589],[5982,5842],[6151,5905],[6212,6020],[5837,6330]]]},
 {"name":"Dankia","color":"#fd00df","id":"b6a0f27c-41fd-4e26-b24a-01acb80a37ba","declutter":3,"polygon":[[[5691,-9770],[5719,-9833],[5701,-9862],[5671,-9865],[5632,-9891],[5532,-9920],[5462,-9919],[5426,-9978],[5419,-10039],[5377,-10074],[5343,-10093],[5326,-10072],[5285,-10056],[5268,-10020],[5286,-9972],[5284,-9945],[5298,-9928],[5290,-9884],[5320,-9872],[5315,-9823],[5335,-9789],[5323,-9775],[5286,-9763],[5280,-9730],[5305,-9710],[5314,-9683],[5340,-9671],[5399,-9671],[5437,-9636],[5458,-9633],[5475,-9591],[5521,-9521],[5714,-9714]]]},
 {"name":"DeltaGroup Northern Exploration Company Private Claims","shortname":"Delta GNEC","color":"#1C7733","id":"c080cfec-05a2-46d0-809d-e81d6dab9fab","declutter":3,"polygon":[[[5377,-10074],[5419,-10039],[5426,-9978],[5462,-9919],[5532,-9920],[5632,-9891],[5671,-9865],[5701,-9862],[5719,-9833],[5839,-9868],[5897,-9939],[5916,-10032],[5858,-10122],[5793,-10144],[5723,-10199],[5742,-10274],[5782,-10290],[5894,-10273],[5945,-10327],[5911,-10369],[5830,-10379],[5768,-10424],[5702,-10376],[5631,-10359],[5570,-10217],[5557,-10211],[5513,-10216],[5428,-10151],[5397,-10186],[5369,-10193],[5344,-10173],[5348,-10100],[5343,-10093]]]},
 {"name":"Delve", "color":"#fcba03","declutter":3,"id":"bba44808-2c43-4cbf-913d-8270bca6076b", "contact": "Troll_Bebek","polygon":[[[-1694,-6073],[-1915,-6073],[-1917,-5844],[-1694,-5844]]]},
+{"name":"Drakonia","color":"#87ceeb","id":"461dcce3-3ade-4307-a281-643822e8af38","polygon":[[[-6351,-2160],[-6360,-2133],[-6357,-2102],[-6321,-2059],[-6321,-2043],[-6320,-2025],[-6309,-2007],[-6320,-1981],[-6292,-1963],[-6261,-1934],[-6231,-1945],[-6190,-1941],[-6158,-1912],[-6126,-1895],[-6126,-1880],[-6137,-1866],[-6132,-1847],[-6126,-1843],[-6148,-1801],[-6143,-1770],[-6148,-1751],[-6110,-1690],[-6092,-1633],[-6061,-1582],[-5842,-1527],[-5638,-1650],[-5568,-1808],[-5466,-1810],[-5330,-1934],[-5260,-1956],[-5260,-2118],[-5260,-2196],[-5390,-2202],[-5501,-2151],[-5573,-2143],[-5616,-2102],[-5668,-2092],[-5700,-2125],[-5724,-2133],[-5750,-2115],[-5802,-2154],[-5814,-2214],[-5835,-2245],[-5860,-2258],[-5892,-2290],[-6046,-2290],[-6173,-2350],[-6234,-2296],[-6363,-2210]]]},
```

Comment:
they quit and relinquished their claims?

Reply:
ah shit. sorry that was deleted in the earlier one i submitted to you - i used an earlier version to make this compilation and it looks like i missed that